File: ezSCORES/ezSCORES.Model/CompetitionsRefereesMatches.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class CompetitionsRefereesMatches
	{
		public int Id { get; set; }

		public int CompetitionsRefereesId { get; set; }

		public int MatchId { get; set; }
		public virtual CompetitionsReferees CompetitionReferee { get; set; }
	}
}

File: ezSCORES/UI/ezscores_mobile/lib/models/competitionsRefereesMatches.dart
import 'package:ezscores_mobile/models/competitionsReferees.dart';
import 'package:json_annotation/json_annotation.dart';

part 'competitionsRefereesMatches.g.dart';

@JsonSerializable()
class CompetitionsRefereesMatches {
	int? id;
	int? competitionsRefereesId;
	int? matchId;
	CompetitionsReferees? competitionReferee;

	CompetitionsRefereesMatches();

	factory CompetitionsRefereesMatches.fromJson(Map<String, dynamic> json) => _$CompetitionsRefereesMatchesFromJson(json);

	Map<String, dynamic> toJson() => _$CompetitionsRefereesMatchesToJson(this);
}